### ISSUE TITLE
ublock-unbreak.txt is out of date

### ISSUE THREAD

**Issue by fmarier** [brave/adblock-lists#52]
Compared to [the upstream ublock unbreak list](https://github.com/uBlockOrigin/uAssets/blob/master/filters/unbreak.txt), [our copy of it](https://github.com/brave/adblock-lists/blob/master/ublock-unbreak.txt) is quite out of date:

    -rw-r--r-- 1 francois francois  24K Feb 26 15:25 ublock-unbreak.txt
    -rw-r--r-- 1 francois francois  94K Feb 27 11:18 unbreak.txt

and likely leads to a lot more breakage in Brave than in uBlock Origin.

I found that our original copy of the ublock-unbreak list (from 53ea03febecb9c3b9e349f101b6a32e4beabd431) is [this one](https://github.com/uBlockOrigin/uAssets/blob/a4abafc0d9d3e57ba92bcda3b3365acdd84d0cf0/filters/unbreak.txt) with one exception:

```
$ diff -u ublock-unbreak.txt unbreak.20161024.txt 
--- ublock-unbreak.txt	2019-02-27 11:28:27.110823480 -0800
+++ unbreak.20161024.txt	2019-02-27 11:29:28.101839184 -0800
@@ -274,7 +274,7 @@
 # - http://www.surfline.com/video/ (links not working)
 # - https://github.com/gorhill/uBlock/issues/529
 # - http://www.scotrail.co.uk/check-your-journey (via https://twitter.com/andy_pee_tho/status/710508529405263872)
-# */b/ss/*&aqe=$image,redirect=1x1-transparent.gif
+*/b/ss/*&aqe=$image,redirect=1x1-transparent.gif
 
 # This unbreak video player:
 # - http://www.cnet.com/videos/big-mac-survives-molten-copper-and-heats-up-internet-ep-233/
```

We commented that rule in our original copy (no hint as to why), but it's never been commented out in the upstream list. It was [added on 2016-03-17](https://github.com/uBlockOrigin/uAssets/commit/36aa3363cf0354790c9c1848836ed5fc2c97d455) and is still in [today's upstream list](https://github.com/uBlockOrigin/uAssets/blob/e3ecf79bfeef19c9571ebba66fb6a20a2c57d16f/filters/unbreak.txt#L268).

Later we upgraded to the [2017-08-28 version](https://github.com/uBlockOrigin/uAssets/blob/eadfa978b67f8f02954698972ce5fead260ab1c6/filters/unbreak.txt
) of the upstream list (in 314de3dd0f223108d66cd5a4ac9c65da4a0eed5b) without any customizations (i.e. dropping the mysterious one above).

In other words, we've pretty much always relied on the upstream unbreak list for this.
